docs/fastdds/dds_layer/topic/instances.rst

```diff
@@ -89,18 +101,180 @@ The following IDL may define the data model of this specific example:
-The instance key is analogous to the primary key in the database (unique identifier of something within the data).
-In this scenario, writing a new sample to the instance is similar to insert/update the database; and disposing the
-instance is like deleting the data corresponding to the given primary key from the database.
+Consider now that the air traffic management center wants to keep a database with the flights being tracked.
+Using DDS instances, maintaining a relational database is almost direct.
+The instance key (unique identifier of the instance) is analogous to the primary key of the database.
+Thus, the airspace traffic management center can keep the latest update for each instance in a table like the one below:
+
+.. list-table::
+    :header-rows: 1
+    :align: left
+
+    * - Instance handle [PK]
+      - Data
+    * - 1
+      - Position1
+    * - 2
+      - Position2
+    * - 3
+      - Position3
+    * - 4
+      - Position4
+    * - 5
+      - Position5
+
+In this case, every time a new sample is received, the corresponding instance entry in the database will be updated with
+the latest known location.
+Disposing the instance may translate in erasing the corresponding data from the database.
+In this scenario, registering and unregistering the instances do not reflect in the database.
+A DataWriter communicating that it is going to be publishing data about a specific instance is of no interest to the
+database until a new data is received and then an insert is directly done with the new discovered instance.
+
+Historical data can also be stored in the relational database, even though depending on the use case, a time series
+database might be considered to improve efficiency.
+In the scenario being considered, the sample timestamp could be used, besides the instance handle, as primary key to
+be able to access the historical tracking data of an specific flight.
+
+.. list-table::
+    :header-rows: 1
+    :align: left
+
+    * - Instance handle [PK]
+      - Source Timestamp [PK]
+      - Data
+    * - 1
+      - 1
+      - Position1
+    * - 2
+      - 1
+      - Position2
+    * - 1
+      - 2
+      - Position3
+    * - 1
+      - 3
+      - Position4
+    * - 2
+      - 2
+      - Position5
+
+In this case, looking for a specific instance handle would return the flight tracking information:
+
+.. list-table::
+    :header-rows: 1
+    :align: left
+
+    * - Instance handle [Fixed]
+      - Source Timestamp
+      - Data
+    * - 1
+      - 1
+      - Position1
+    * - 1
+      - 2
+      - Position3
+    * - 1
+      - 3
+      - Position4
+
+Whereas looking for a specific timestamp would allow to have a picture of the different flight locations at a specific
+time:
+
+.. list-table::
+    :header-rows: 1
+    :align: left
+
+    * - Instance handle
+      - Source Timestamp [Fixed]
+      - Data
+    * - 1
+      - 2
+      - Position3
+    * - 2
+      - 2
+      - Position5
```

Comment:
I'd add a final sentence stating that dispose/unregister may mean the same as in te previous DB example

Reply:
I do not think is needed. It is not properly another example but another case of the same example. So the information given previously is still valid in this second scenario.